suite/tests/CMakeLists.txt

```diff
@@ -2380,23 +2400,25 @@ if (CLIENT_INTERFACE)
         # more deterministic output.
         torunonly_ci(tool.drcachesim.threads client.annotation-concurrency drcachesim
           "drcachesim-threads.c" # for templatex basename
-          "-ipc_name drtestpipe2" "" "${annotation_test_args}")
+          # XXX: the threading pool seems slower than synched write-out
+          "-ipc_name drtestpipe2 -num_threads 0" "" "${annotation_test_args}")
         set(tool.drcachesim.threads_toolname "drcachesim")
         set(tool.drcachesim.threads_basedir
           "${PROJECT_SOURCE_DIR}/clients/drcachesim/tests")
         set(tool.drcachesim.threads_rawtemp ON) # no preprocessor
-        set(tool.drcachesim.threads_timeout 150) # This test is long.
+        set(tool.drcachesim.threads_timeout 200) # This test is long.
```

Comment:
The pool slows all of these down so much we need longer timeouts??  That's not good.  Sounds like this should definitely be off by default until its performance is improved.